draft-ietf-rtcweb-jsep.xml

```diff
@@ -1521,18 +1521,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>The next step is to generate m= sections, as specified in
           <xref target="RFC4566"/> Section 5.14, for each
-          MediaStreamTrack that has been added to the PeerConnection via the
-          addStream method. (Note that this method takes a MediaStream, which
-          can contain multiple MediaStreamTracks, and therefore multiple m=
-          sections can be generated even if addStream is only called once.)
-          m=sections MUST be sorted first by the order in which the
-          MediaStreams were added to the PeerConnection, and
-          then by the alphabetical ordering of the media type for the MediaStreamTrack.
-          For example, if a MediaStream containing both an audio and a video
-          MediaStreamTrack is added to a PeerConnection, the resultant m=audio
-          section will precede the m=video section. If a second MediaStream
-          containing an audio MediaStreamTrack was added, it would follow
-          the m=video section.</t>
+          RtpTransceiver that has been added to the PeerConnection via
+          the addTrack or addTransceiver methods, excluding stopped
+          RtpTransceivers. m= sections are generated in the order that
```

Comment:
Yes, in #215, I change it, but failed to push in time, to say "excluding stopped RtpTranscievers that are not associated with any m= section (either due to m= section recycling or having been stopped before being associated with an m= section)".  I'll change it to that.